apps/website/src/examples/Stepper.layout.tsx

```diff
@@ -0,0 +1,100 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Button, Flex, Input, Label, Stepper, InputGroup, Radio } from '@itwin/itwinui-react';
+
+const stepLabels = [{ name: 'User Info' }, { name: 'Color Selection' }, { name: 'Explanation' }];
+
+export default () => {
+  const [currentStep, setCurrentStep] = React.useState(0);
+  const [disableProgress, setDisableProgress] = React.useState(true);
+
+  React.useEffect(() => {
+    setDisableProgress(true);
```

Comment:
This can be moved into `onStepClick` instead of `useEffect`

Reply:
Fixed